### PR TITLE
feat(choose-UI): Add capabilities to choose a different UI

### PR DESCRIPTION
**Pull request by labdala**
**Under construction**

Adds the possibility of choosing a different UI (e.g., itk-viewer-bootstrap-ui) to the itkwidgets version 1.0a5.

### CODE REVIEW

File: itkwidgets/viewer.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
 class ViewerRPC:
     """Viewer remote procedure interface."""
 
-    def __init__(self, ui_collapsed=True, rotate=False, **add_data_kwargs):
+    def __init__(self, ui_collapsed=True, rotate=False, UI='pydata-sphinx', **add_data_kwargs):
```
Comment (thewtex) [inline itkwidgets/viewer.py:19]: `UI` -> `ui`

File: itkwidgets/viewer.py [hunk 2]
```diff
@@ -35,21 +34,33 @@ async def setup(self):
...
         global _viewer_count
+        ui = self._init_viewer_kwargs.get('config', None)
+        if ui=='pydata-sphinx':
+            config = { 'uiMachineOptions': { 'href': 'https://cdn.jsdelivr.net/npm/itk-viewer-bootstrap-ui@0/dist/itk-viewer-bootstrap-ui.es.js', 'export': 'default' }}
+        elif ui=='mui':
+            config = { 'uiMachineOptions': { 'href': 'https://www.jsdelivr.com/package/npm/itk-viewer-material-ui@0/dist/materialUIMachineOptions.js.es.js', 'export': 'default' }}
+        elif ui!='reference':
+            config=ui
+        else:
+             config={}
```
Comment (thewtex) [inline itkwidgets/viewer.py:45]: Please run `black` for styling

File: itkwidgets/viewer.py [hunk 3]
```diff
@@ -65,9 +76,9 @@ def set_default_ui_values(self, itk_viewer):
 class Viewer:
     """Pythonic Viewer class."""
 
-    def __init__(self, ui_collapsed=True, rotate=False, **add_data_kwargs):
+    def __init__(self, ui_collapsed=True, rotate=False, ui='reference', **add_data_kwargs):
```
Comment (thewtex) [inline itkwidgets/viewer.py:79]: `pydata-sphinx` as the default

**Review by thewtex — CHANGES_REQUESTED** [review]

File: itkwidgets/viewer.py [hunk 4]
```diff
@@ -35,21 +34,33 @@ async def setup(self):
     async def run(self, ctx):
         """ImJoy plugin setup function."""
         global _viewer_count
+        ui = self._init_viewer_kwargs.get('config', None)
```
Comment (thewtex) [inline itkwidgets/viewer.py:37]: 'config' -> 'ui'

File: itkwidgets/viewer.py [hunk 5]
```diff
@@ -65,9 +76,9 @@ def set_default_ui_values(self, itk_viewer):
 class Viewer:
     """Pythonic Viewer class."""
 
-    def __init__(self, ui_collapsed=True, rotate=False, **add_data_kwargs):
+    def __init__(self, ui_collapsed=True, rotate=False, ui='reference', **add_data_kwargs):
         """Create a viewer."""
-        self.viewer_rpc = ViewerRPC(ui_collapsed=ui_collapsed, rotate=rotate, **add_data_kwargs)
+        self.viewer_rpc = ViewerRPC(ui_collapsed=ui_collapsed, rotate=rotate, config=ui, **add_data_kwargs)
```
Comment (thewtex) [inline itkwidgets/viewer.py:81]: 'config' -> 'ui'

File: itkwidgets/integrations/__init__.py [hunk 6]
```diff
@@ -11,25 +11,25 @@
 
 _image_count = 1
 
-async def _set_viewer_image(itk_viewer, image, name=None):
+async def _set_viewer_image(itk_viewer, image, config, name=None):
```
Comment (thewtex) [inline itkwidgets/integrations/__init__.py:14]: remove `config`

File: itkwidgets/viewer.py [hunk 7]
```diff
@@ -35,21 +34,33 @@ async def setup(self):
...
             name =f'itkwidgets viewer {_viewer_count}',
             type='itk-vtk-viewer',
             src='https://kitware.github.io/itk-vtk-viewer/app',
             fullscreen=True,
+             #config should be a python data dictionary and can't be a string e.g. 'pydata-sphinx',
+            config = config,
         )
         _viewer_count += 1
 
         data, input_type = self._get_input_data()
```
Comment (thewtex) [inline itkwidgets/viewer.py:57]: move this above `createWindow` and pass in `data` to `ctx`

**Review by thewtex — CHANGES_REQUESTED** [review]

**Comment by labdala** [comment]
Changes addressed in #482.

**Pull request closed without merging**